plugins/plugin-kiali/src/components/panel/Graph.tsx

```diff
@@ -0,0 +1,300 @@
+// See: https://js.cytoscape.org/#style
+const styleSheet: cytoscape.Stylesheet[] = [
+  {
+    selector: 'node',
+    style: {
+      'background-color': '#ffffff',
+      'background-fit': 'cover',
+      'background-height': '80%',
+      'background-image': 'data(nodeImage)',
+      'background-position-x': '50%',
+      'background-position-y': '50%',
+      'background-width': '80%',
+      'border-color': '#6a6e73',
+      'border-opacity': 1,
+      'border-width': 1,
+      color: '#151515',
+      'font-family': 'RedHatDisplay, Overpass, overpass, helvetica, arial, sans-serif',
+      // The label is only rendered, so that the generated html labels fit the nodes. We disable the default label, via
+      // the text-opacity setting.
+      label: 'data(nodeLabelFull)',
+      'text-halign': 'center',
+      'text-opacity': 0,
+      'text-valign': 'bottom',
+      'text-wrap': 'wrap',
+    },
+  },
+  {
+    selector: 'node[nodeType="box"]',
+    style: {
+      shape: 'roundrectangle',
+    },
+  },
+  {
+    selector: 'node[nodeType="app"]',
+    style: {
+      shape: 'roundrectangle',
+    },
+  },
+  {
+    selector: 'node[nodeType="service"]',
+    style: {
+      'background-position-y': '6px',
+      shape: 'round-triangle',
+    },
+  },
+  {
+    selector: 'node[nodeType="serviceentry"]',
+    style: {
+      shape: 'round-tag',
+    },
+  },
+  {
+    selector: 'node[nodeType="unknown"]',
+    style: {
+      shape: 'ellipse',
+    },
+  },
+
+  {
+    selector: 'edge',
+    style: {
+      color: '#151515',
+      'curve-style': 'bezier',
+      'font-family': 'RedHatDisplay, Overpass, overpass, helvetica, arial, sans-serif',
+      'font-size': 10,
+      label: 'data(edgeLabel)',
+      'line-color': '#6a6e73',
+      'target-arrow-color': '#6a6e73',
+      'target-arrow-shape': 'triangle',
+      'text-wrap': 'wrap',
+      width: 3,
+    },
+  },
+  {
+    selector: 'edge[edgeType="tcp"]',
+    style: {
+      'line-color': '#0066cc',
+      'target-arrow-color': '#0066cc',
+    },
+  },
+  {
+    selector: 'edge[edgeType="grpc"]',
+    style: {
+      'line-color': '#009596',
+      'target-arrow-color': '#009596',
+    },
+  },
+  {
+    selector: 'edge[edgeType="http"]',
+    style: {
+      'line-color': '#3e8635',
+      'target-arrow-color': '#3e8635',
+    },
+  },
+  {
+    selector: 'edge[edgeType="httphealthy"]',
+    style: {
+      'line-color': '#3e8635',
+      'target-arrow-color': '#3e8635',
+    },
+  },
+  {
+    selector: 'edge[edgeType="httpdegraded"]',
+    style: {
+      'line-color': '#f0ab00',
+      'target-arrow-color': '#f0ab00',
+    },
+  },
+  {
+    selector: 'edge[edgeType="httpfailure"]',
+    style: {
+      'line-color': '#c9190b',
+      'target-arrow-color': '#c9190b',
+    },
+  },
+];
+
+const nodeLabel = (node: INodeData): string => {
+  if (node.isBox) {
+    return `<div class="kobsio-kiali-label boxed">
```

Comment:
Can you add a css file with the following content and import it into this component, If this is missing the styling of the nodes / edges is not correct.

```css
/* kobsio-kiali-label
 * The kobsio-kiali-label classes are used for the node labels in the Kiali graph. */
 .kobsio-kiali-label {
  border-radius: 3px;
  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 2px 8px 0 rgba(0, 0, 0, 0.19);
  display: flex;
  font-family: Verdana,Arial,Helvetica,sans-serif,pficon;
  font-size: 8px;
  font-weight: normal;
  line-height: 11px;
  margin-top: 4px;
  text-align: center;
}

.kobsio-kiali-label.boxed {
  margin-top: 13px;
}

.kobsio-kiali-label-text {
  align-items: center;
  background-color: var(--pf-global--palette--white);
  border-radius: 3px;
  border-width: 1px;
  color: var(--pf-global--palette--black-1000);
  display: flex;
  font-size: 8px;
  padding: 3px 5px;
}

.kobsio-kiali-label-text.boxed {
  background-color: var(--pf-global--palette--black-700);
  color: var(--pf-global--palette--white);
}

.kobsio-kiali-label-text.icon {
  border-bottom-left-radius: unset;
  border-color: var(--pf-global--palette--purple-400);
  border-left: 0;
  border-style: solid;
  border-top-left-radius: unset
}

.kobsio-kiali-label-badge {
  background-color: var(--pf-global--primary-color--100);
  margin-right: 5px;
  min-width: 24px;
  padding-left: 0px;
  padding-right: 0px;
}

.kobsio-kiali-label-icon-wrapper {
  align-items: center;
  background-color: var(--pf-global--palette--purple-400);
  border-bottom-left-radius: 3px;
  border-top-left-radius: 3px;
  color: var(--pf-global--palette--white);
  display: flex;
  font-size: 12px;
  padding: 3px 3px;
}

.kobsio-kiali-label-icon {
  margin-left:1px;
}
```